crates/bevy_editor_settings/src/local_prefs.rs

```diff
@@ -0,0 +1,30 @@
+use bevy::{prelude::Resource, reflect::Reflect};
+
+use crate::{SettingsTags, SettingsType};
+
+
+
+#[derive(Debug, Clone, PartialEq, Eq, Reflect, Resource)]
+#[reflect(@SettingsType::Project, @SettingsTags(vec!["basic", "settings", "testing"]))]
+/// Settings for building the project
+pub struct Build {
+    /// The Command for building the project in debug mode
+    debug: String,
+    /// The Command for building the project in release mode
+    release: String,
+    /// The Command for running the project in debug mode
+    run_debug: String,
+    /// The Command for running the project in release mode
+    run_release: String,
+}
+
+impl Default for Build {
+    fn default() -> Self {
+        Self {
+            debug: "cargo build".to_string(),
+            release: "cargo build --release".to_string(),
+            run_debug: "cargo run".to_string(),
+            run_release: "cargo run --release".to_string(),
+        }
+    }
+}
```

Comment:
Shouldn't we just generalize this as "profiles" or "tasks"?
Local builds can be different than CI builds, also we could use profiles for various tests

Since we're hardcoding this right now anyways, we might as well do
```rs
Profile {
  name: "Debug Build".to_string(),
  command: "cargo build".to_string(),
  description: None,
}
```
and store it in a file

Reply:
sorry meant to reply to this. that specific code was put together in like two mins. but the idea is you can override the build commands in a toml file at the root of the project. so the editor know how to start the game no matter the build system.

Reply:
btw not against your changes but would you be able to explain why the editor needs to know how to build in the app in ci?

Reply:
Editor should provide basic CLI for automation, even if it's technically another app, it's within the scope of editor as a project

Reply:
i think i will remove all this for this pr anyways. it was kinda based on my original design i went away from. it should be super easy to add back in with some more thought